.pubnub.yml

```diff
@@ -87,6 +87,22 @@ features:
     - PUBLISH-ASYNC
     - PUBLISH-REPLICATION-FLAG
     - PUBLISH-MESSAGE-TTL
+  subscribe:
+    - SUBSCRIBE-CHANNELS
+    - SUBSCRIBE-CHANNEL-GROUPS
+    - SUBSCRIBE-PRESENCE-CHANNELS
+    - SUBSCRIBE-PRESENCE-CHANNELS-GROUPS
+    - SUBSCRIBE-WITH-TIMETOKEN
+    - SUBSCRIBE-FILTER-EXPRESSION
+    - SUBSCRIBE-PUBLISHER-UUID
+    - SUBSCRIBE-PUBSUB-V2
+    - SUBSCRIBE-MESSAGE-ACTIONS-LISTENER
```

Comment:
@Xavrax if this one about separate listeners - we don't have separate listener for actions.
If it is about listener's ability to inform about those events, then in addition we have:
* `SUBSCRIBE-SIGNAL-LISTENER`
* `SUBSCRIBE-MEMBERSHIP-LISTENER`
* `SUBSCRIBE-OBJECTS-CHANNEL-LISTENER`
* `SUBSCRIBE-OBJECTS-UUID-LISTENER`
* `SUBSCRIBE-FILE-LISTENER`

Reply:
DM'd you